frontend/app/src/entities/nodes/relationships/ui/object-tab-manager.tsx

```diff
@@ -0,0 +1,72 @@
+import { TASK_TAB } from "@/config/constants";
+import { QSP } from "@/config/qsp";
+import { ObjectRelationshipsManager } from "@/entities/nodes/relationships/ui/object-relationships-manager";
+import { REPOSITORY_OBJECTS_TAB } from "@/entities/repository/constant";
+import { RepositoryObjectsManager } from "@/entities/repository/ui/repository-objects-manager";
+import { ModelSchema } from "@/entities/schema/types";
+import { TaskItemDetails } from "@/entities/tasks/ui/task-item-details";
+import { TaskItems } from "@/entities/tasks/ui/task-items";
+import { Node } from "@/shared/api/graphql/generated/graphql";
+import { constructPath } from "@/shared/api/rest/fetch";
+import { Link } from "@/shared/components/ui/link";
+import { Icon } from "@iconify-icon/react";
+import { useRef } from "react";
+import { StringParam, useQueryParam } from "use-query-params";
+
+export interface ObjectTabManagerProps {
+  parentNodeSchema: ModelSchema;
+  objectDetailsData: Node;
+  relationshipName: string;
+}
```

Comment:
- `relationshipName` is not used
- `parentNodeSchema` and `objectDetailsData` names are confusing because there are related but the name suggest it is different. `objectSchema` is good enough imo
- `objectDetailsData` type is more precise with NodeObject